src/ButterCMS/ButterCMS.php

```diff
@@ -27,14 +27,18 @@ public function __construct($authToken)
         $this->client = new Client();
     }
 
-    protected function request($url, $params = [])
+    protected function request($url, $params = [], $tryCount = 0)
     {
         try {
             $params['auth_token'] = $this->authToken;
             $response = $this->client->get(self::API_ROOT_URL . $url, [
                 'query' => $params
             ]);
         } catch (ClientException $e) {
+            if ($tryCount < 1) {
+                return $this->request($url, $params, ++$tryCount);
+            }
+
             return false;
```

Comment:
@joshholat Should this re-raise the exception instead of returning `false`? Otherwise the consumer wouldn't know that an exception is occurring and instead would just get a non-useful return result.